modules/steam.nix

```diff
@@ -403,6 +415,9 @@ in
         # Enable dynamic backlight, we have the kernel patch to disable events
         STEAM_ENABLE_DYNAMIC_BACKLIGHT = "1";
 
+        # Enabled fan control toggle in steam
+        STEAM_ENABLE_FAN_CONTROL = "1";
+
```

Comment:
I don't know what this adds. Vendor added this since I last looked, no obvious toggles in Steam. Might be dead code.